WordPress/build.gradle

```diff
@@ -51,7 +50,6 @@ android {
 
     productFlavors {
         vanilla { // used for release and beta
-            buildConfigField "boolean", "AZTEC_EDITOR_AVAILABLE", "false"
```

Comment:
Should we drop `AZTEC_EDITOR_AVAILABLE` (always true) and `AppPrefs.isAztecEditorAvailable()` (always true)?